src/js/lib/utilities.js

```diff
@@ -20,3 +20,13 @@ export function replaceNodeWith(oldNode, newNode) {
 
     return oldNode.parentNode.replaceChild(newNode, oldNode);
 }
+
+/**
+ * Removes trailing/leading whitespace from a value and makes it lowercase.
+ * @memberof module:CustomUi
+ * @param  {String} [value=''] String to be formatted.
+ * @return {String}             Formatted value.
+ */
+export function formatValue(value) {
```

Comment:
I still feel formatValue is a little too general. Since we're only formatting strings formatStrings makes a lot of sense to me. :P